### PR TITLE
8634 endpoint security override is not being respected

### PR DESCRIPTION
**Pull request by david-martin**

### CODE REVIEW

File: application.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ var cors = require('cors');
 // list the endpoints which you want to make securable here
 var securableEndpoints;
 // fhlint-begin: securable-endpoints
-securableEndpoints = ['hello'];
+securableEndpoints = ['/hello'];
```
Comment (david-martin) [inline application.js]: actual fix is here.
rest is :lipstick:

**Comment by jasonmadigan** [comment]
Looks good to me :arrow_up:

**Comment by cianclarke** [comment]
:point_left:

**Comment by jasonmadigan** [comment]
absolutely :racehorse:

**Comment by cianclarke** [comment]
:dog2: (doge side profile)

**Pull request merged**